mobile/library/cc/engine_builder.cc

```diff
@@ -84,8 +84,8 @@ EngineBuilder& EngineBuilder::setOnEngineRunning(std::function<void()> closure)
   return *this;
 }
 
-EngineBuilder& EngineBuilder::addStatsSink(std::string name, std::string typed_config) {
-  stats_sinks_.push_back(std::make_pair(name, typed_config));
+EngineBuilder& EngineBuilder::addStatsSinks(const std::vector<std::string>& stats_sinks) {
+  stats_sinks_ = std::move(stats_sinks);
```

Comment:
nit: since we're doing a move here it would be better to pass the argument by value not by const-ref.